src/models/User.js

```diff
@@ -19,6 +19,10 @@ const UserSchema = new Schema(
       enum: ["instructor", "admin", "revoked"],
       required: [true, "Access Level is required"],
     },
+    instructor: {
```

Comment:
I think this needs to be removed like it was in the query-files branch. Instructors store a reference to their associated user, not the other way around.

Reply:
That was my original thought as well. After discussions with Justin this has been altered so that a user such as a teacher assistant would reference an instructor.

Reply:
@jcypret could you confirm? If that's the case, then ignore my comment above on the user seeder.